pgbot/routine.py

```diff
@@ -119,6 +127,8 @@ async def routine():
     if common.guild is not None:
         async with db.DiscordDB("reminders") as db_obj:
             await handle_reminders(db_obj)
+        async with db.DiscordDB("bot_mutes") as db_obj:
```

Comment:
This part can be unindented by a level, we do not want this check under `common.guild is None` case